### PR TITLE
Add a program to evaluate a string in prefix notation (Polish Notation)

### PR DESCRIPTION
**Pull request by M0r13n**
We already have a script for evaluating the postfix notation. So I added another for prefix notation.

### CODE REVIEW

File: data_structures/stacks/prefix_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+        # push operand to stack
+        if is_operand(c):
+            stack.append(int(c))
+
+        else:
+            # pop values from stack can calculate the result
+            # push the result onto the stack again
+            o1 = stack.pop()
+            o2 = stack.pop()
```
Comment (onlinejudge95) [inline data_structures/stacks/prefix_evaluation.py]: What if there is only a single input in the string
`evaluate("9")`
A better error message would be useful.

Comment (M0r13n) [reply data_structures/stacks/prefix_evaluation.py]: I think the current behaviour is mathematically correct. Just like in infix notation x = 9 is a valid expression. At least I think so. So `evaluate("9")` should output "9" as it does.

Comment (onlinejudge95) [reply data_structures/stacks/prefix_evaluation.py]: Early morning sleep made me skip the entire logic :p

Comment (cclauss) [reply data_structures/stacks/prefix_evaluation.py]: ☕️

**Pull request merged**